### ISSUE TITLE
ERROR: Invalid command: None)

### ISSUE THREAD

**Issue by ofrei** [unioslo/tsd-s3cmd#4]
I'm getting the following error:

```
tsd-s3cmd mb s3://ofrei
User > p697-ofrei
Password > 
OTP > 022919
ERROR: Invalid command: None)
```

My environment: I've trying clean installation on my laptop, connected to UiO network via VPN.  I've re-installed tsd-s3cmd today by downloading .zip file from latest master batch of this repo (latest commit https://github.com/unioslo/tsd-s3cmd/commit/f616965bba26d475fabcbfa58e18bc3de154168b ), and installed it with ``pip3 install tsd-s3cmd-master.zip``. 
I succesfully  registered it to p697 project. Then I'm getting ``ERROR: Invalid command: None)`` , apparently from all commands (at least from ``sync`` and ``mb``).

**Comment by haatveit** [comment]
Thank you for reporting this issue.

There was a breaking change to authentication in the most recent version of tsd-api-client. I've pushed a simple fix in f70f855 and bumped the package version for a new release:

https://github.com/unioslo/tsd-s3cmd/releases/tag/v0.1.3

**Comment by ofrei** [comment]
Thanks for fixing this! I've updated tsd-s3cmd and it's working fine.
